File: programacao/c/respostas/operacoes/35.c
#include <stdio.h>

int main() {
    float A, B, C;
    float X, Y, Z;
    int cont_inteiras;

    printf("Digite a largura do conteiner (A): ");
    scanf("%f", &A);

    printf("Digite o comprimento do conteiner (B): ");
    scanf("%f", &B);

    printf("Digite a altura do conteiner (C): ");
    scanf("%f", &C);

    printf("Digite a largura do armazem (X): ");
    scanf("%f", &X);

    printf("Digite o comprimento do armazem (Y): ");
    scanf("%f", &Y);

    printf("Digite o pe direito do armazem (Z): ");
    scanf("%f", &Z);

    int cont_largura = (int)(X / A);
    int cont_comprimento = (int)(Y / B);
    int cont_altura = (int)(Z / C);

    cont_inteiras = cont_largura * cont_comprimento * cont_altura;

    printf("A quantidade maxima de conteineres e: %d\n", cont_inteiras);

    return 0;
}

File: programacao/python/respostas/operacoes/35.py
A = float(input("Largura do conteiner (A): "))
B = float(input("Comprimento do conteiner (B): "))
C = float(input("Altura do conteiner (C): "))
X = float(input("Largura do armazem (X): "))
Y = float(input("Comprimento do armazem (Y): "))
Z = float(input("Pe direito do armazem (Z): "))

cont_largura = X // A
cont_comprimento = Y // B
cont_altura = Z // C

total = cont_largura * cont_comprimento * cont_altura
print(f"A quantidade maxima de conteineres e: {total}")
